functions/source/EksClusterResource/.rpdk-config

```diff
@@ -1,9 +1,9 @@
 {
     "typeName": "AWSQS::EKS::Cluster",
     "language": "go",
-    "runtime": "go1.x",
-    "entrypoint": "handler",
-    "testEntrypoint": "handler",
+    "runtime": "provided.al2023",
+    "entrypoint": "bootstrap",
```

Comment:
entrypoint must match module name, so I think it should be `resource` not bootstrap

Reply:
Reference: https://aws.amazon.com/blogs/compute/migrating-aws-lambda-functions-from-the-go1-x-runtime-to-the-custom-runtime-on-amazon-linux-2/

> Note: The handler value is not used by the provided.al2 runtime, nor the aws-lambda-go library, and may be set to any value. We recommend the setting the value to bootstrap to help with migrating between go1.x and provided.al2.

Reply:
Reference: https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/runtime-update.html#os-only-runtime-upgrade

## Upgrading the Go runtime
**Steps to upgrade to provided.al2**
1. In the .rpdk-config file:
   * Change the runtime parameter to porovided.al2.
   * Change entrypoint and testEntrypoint paramters to bootstrap.
   
    The following is an example .rpdk-config file using the provided.al2 runtime:
```
{
    "typeName": "Example::GitHub::Repo",
    "language": "go",
    "runtime": "provided.al2",
    "entrypoint": "bootstrap",
    "testEntrypoint": "bootstrap",
    "settings": {
        "import_path": "github.com/aws-cloudformation/cloudformation-cli-go-plugin/examples/github-repo",
        "protocolVersion": "2.0.0",
        "pluginVersion": "2.0.0"
    }
}
```

2. In the template.yml file:
   * In TypeFunction and TestEntrypoint, change the Runtime parameters to provided.al2.
   * In TypeFunction and TestEntrypoint, change the Handler parameters to bootstrap.
   * In TypeFunction and TestEntrypoint, add x86_64 to the Architecture parameter.